### ISSUE TITLE
[Content styleguide] The content isn't written consistently across static pages, tools, and devices.  (05.11.1)

### ISSUE THREAD

**Issue by chaseakirby** [department-of-veterans-affairs/va.gov-team#39475]
#### VFS team name
Authenticated Experience

#### VFS product name
My VA Dashboard

#### VFS feature name
Payment Information

#### Point of Contact/Reviewers
Cheryl Evans (@CherylEvans) - Content

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
The content isn't written consistently across static pages, tools, and devices. 

### Issue Details
Payments/Debts are grouped together, and placed one under the other on the page, yet when Payments amounts are listed, the font appears larger and the heading is not consistent to font/heading/style when Debts exist. 

### VA.gov Experience Standard
[Category Number 05, Issue Number 11](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.3.2_A

---

### Platform Recommendation
Since Payments/Debts are grouped together in the same hierarchal plane, they should be treated the same with respect to placement/font/style.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by hdjustice** [comment]
4/6/22 - Discussion with Angela & Samara - We are working on the organization of the section for future iterations

**Comment by shiragoodman** [comment]
@ajakabcin has this issue been resolved? This is a `launch-blocking` issue and should be considered high priority.

**Comment by aagosto90** [comment]
This feedback is for the previous version of payment info "Benefit Payments and Debts" which is no longer live and has been replaced by separate sections for Outstanding Debts and Benefit Payments. We've handled all must feedback for the new version prior to launch so I think this ticket can be closed @shiragoodman @ajakabcin @ericasauv09

**Comment by shiragoodman** [comment]
Thank you @aagosto90 - I think we can consider this resolved and I'll close the ticket.
